tools/NodeConfigCompare/configuration/cloud_formation_template.yaml

```diff
@@ -968,49 +909,76 @@ Resources:
+                file.retrieve(url, f'/tmp/{filename}')
+              
+              files = glob.glob('*')
+              try:
+                for file in files:  
+                  s3_client.meta.client.upload_file(file, Bucket, Key=f'{path}/{file}')
+                cfnresponse.send(event, context, cfnresponse.SUCCESS, {'Data': 'Copy Succeeded'})
+              except Exception as e:
+                print(e)
+                cfnresponse.send(event, context, cfnresponse.FAILED, {'Data': 'Copy Failed'})
+
   CopyRedshiftConfigTestingCode:
     Type: Custom::CopyRedshiftConfigTestingCode
     Properties:
       ServiceToken:
-        Fn::GetAtt : [LambdaFunctionS3Copy, Arn]
-      source_folder_s3_path: !Sub
-        - s3://${ExternalPublicBucket}/${ScriptPrefix}
-        - {ExternalPublicBucket: !FindInMap [ ConfigTesting, S3, ExternalPublicBucket],ScriptPrefix: !FindInMap [ ConfigTesting, S3, ScriptPrefix]}
+        Fn::GetAtt : [LambdaFunctionGitHubCopy, Arn]
+      URLs:  [
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/python_scripts/RedshiftConfigTestingLambda.py.zip",
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/python_scripts/StartUpLambda.py.zip",
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/python_scripts/boto3-redshift-serverless.zip",
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/python_scripts/create_external_schema.py",
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/python_scripts/redshift-performance-test.py",
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/python_scripts/python.zip",
+
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/bootstrap_scripts/extract_bootstrap.sh",
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/bootstrap_scripts/performance_test_bootstrap.sh",
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/bootstrap_scripts/replay_bootstrap.sh",
+
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/configuration/RedshiftConfigTestingStepFunction.json",
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/configuration/parameter_group_config.json",
+
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/sql/gather_comparison_stats.sql",
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/sql/gather_comparison_stats_serverless.sql",
+        "https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/sql/populate_comparison_results.sql"
```

Comment:
There appears to be a lot of repetitive strings here, why not just use the file names with folder enclosing it?

Reply:
I am planning to add a line in the lambda like this in the first for loop 

```
    url = 'https://raw.githubusercontent.com/aws/redshift-test-drive/main/tools/NodeConfigCompare/'+ file_path

```

and in the lines 960 to 976 i will have this
```
"URLs":[
                "python_scripts/RedshiftConfigTestingLambda.py.zip"
                ],
```

Let me know if this is what you are looking for ?

Reply:
Yup this works, just to be a bit more flexible, do you think we can take the `branch` as an argument as well so that we can have customers switch to a different branch if needed?

Reply:
yes, Ill add it if I can. Thanks

Reply:
Done, 
Tested the CFT template and resources were successfully created.

Test stack name in CloudFormation: T28-NodeConfigRevamp